utilities/pipelines/resourceRemoval/helper/Remove-Deployment.ps1

```diff
@@ -90,8 +90,8 @@ function Remove-Deployment {
 
         # Pre-Filter & order items
         # ========================
-        [array] $rawResourceIdsToRemove = $deploymentResourceIds | Sort-Object -Property { $_.Split('/').Count } -Descending | Select-Object -Unique
-        Write-Verbose ('Total number of deployments after pre-filtering & ordering items [{0}]' -f $rawResourceIdsToRemove.Count) -Verbose
+        $rawResourceIdsToRemove = $deploymentResourceIds | Sort-Object -Property { $_.Split('/').Count } -Descending | Select-Object -Unique
```

Comment:
Are changes to resource removal intended?

Reply:
see other comment